client/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:12
+
+# make the 'app' folder the current working directory
+WORKDIR /app
+
+# copy both 'package.json' and 'package-lock.json' (if available)
+COPY package*.json ./
+
+# install project dependencies
+RUN npm install
+
+# copy project files and folders to the current working directory (i.e. 'app' folder)
+COPY . .
+
+EXPOSE 8080
+CMD [ "npm", "run", "serve" ]
```

```suggestion
CMD [ "npm", "run", "serve" ]
```
Add breakline to end of file